NOnion/Utility/DateTimeUtils.fs

```diff
@@ -8,3 +8,6 @@ module DateTimeUtils =
 
     let internal ToUnixTimestamp(dt: DateTime) =
         (GetTimeSpanSinceEpoch dt).TotalSeconds |> uint
+
+    let internal FromUnixTimestamp(num: uint) =
+        num |> float |> DateTime(1970, 1, 1).AddSeconds
```

Comment:
DateTime(1970, 1, 1) is now hardcoded twice here, let's put it in a private constant